src/test/java/racingcar/domain/CarsTest.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import racingcar.strategy.RandomValueStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarsTest {
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 1, 2, 3, 4})
+    @DisplayName("입력한 차 대수만큼 차 리스트를 생성을 확인하는 테스트")
+    void carListCreateTest_thenIsEqualTo(int carCount) {
+        List<Car> carList = new ArrayList<>();
+        for (int i = 0; i < carCount; i++) {
+            carList.add(new Car());
+        }
+        assertThat(new Cars(carCount)).isEqualTo(new Cars(carList));
```

Comment:
```suggestion
        List<Car> carList = new ArrayList<>();
        for (int i = 0; i < carCount; i++) {
            carList.add(new Car());
        }

        assertThat(new Cars(carCount)).isEqualTo(new Cars(carList));
```

assertion 전에 한 줄을 띄워 주면 무엇을 검증하는지 눈에 잘 들어오죠. 빈 줄을 잘 쓰는 것도 가독성에 아주 중요하답니다.